Rewrite and split the Overview article (#2834)
surveyjs/survey-creator: docs/Survey-Creator-Overview.md

# Survey Creator / Form Builder Overview

Survey Creator / Form Builder is a visual designer that allows users to create surveys and forms. The designer produces a survey configuration in the JSON format. This configuration can be later used by the SurveyJS library to display the survey or form in your application.

Survey Creator / Form Builder ships with the following features:

- Native support for React and Knockout
- Integration of the Knockout version into Angular, Vue, and jQuery applications
- Modern, customizable UI themes
- TypeScript support
- [Community-supported UI localization to 25+ languages](https://surveyjs.io/Documentation/Survey-Creator?id=localization)
- [Integration with any backend framework](https://surveyjs.io/Documentation/Survey-Creator?id=integration-with-backend) (examples for PHP, NodeJS, and ASP.NET included)
- Survey JSON definitions can be stored in any database

> - [3rd-party component integration](https://surveyjs.io/Documentation/Survey-Creator?id=Create-Custom-Widget)

Reviewer: ```suggestion
- [Third-party component integration](https://surveyjs.io/Documentation/Survey-Creator?id=Create-Custom-Widget)
```

[Try Survey Creator / Form Builder](https://surveyjs.io/create-survey)

Survey Creator is **not available for free commercial usage**. If you want to integrate it into your application, you must purchase a [commercial license](https://surveyjs.io/Licenses#SurveyCreator). However, you can use [online Survey Creator](https://surveyjs.io/create-survey) to produce JSON survey configurations and use them with SurveyJS in your application free of charge.

For information on how to add Survey Creator / Form Builder to your application, refer to one of the following Get Started help topics:

- [Angular](https://surveyjs.io/Documentation/Survey-Creator?id=get-started-angular)
- [Vue](https://surveyjs.io/Documentation/Survey-Creator?id=get-started-vue)
- [React](https://surveyjs.io/Documentation/Survey-Creator?id=get-started-react)
- [Knockout / jQuery](https://surveyjs.io/Documentation/Survey-Creator?id=get-started-knockout-jquery)

> We also provide over [40 demo examples](https://surveyjs.io/Examples/Survey-Creator) that you can edit and copy code from.

Reviewer: ```suggestion
We also include over [40 demo examples](https://surveyjs.io/Examples/Survey-Creator) that allow you to edit and copy code.
```
